### PR TITLE
SUIT-10092 - Add CodeClimate badges to JS API repo

### PR DESCRIPTION
**Pull request by andriichumak**
After this merge we'll start to respect what CodeClimate suggest and will no longer merge PRs if CodeClimate maintainability decreases with your changes.

### CODE REVIEW

File: .codeclimate.yml [hunk 1]
```diff
@@ -1,3 +1,35 @@
...
+checks:
+  argument-count:
+    config:
+      threshold: 4
+  complex-logic:
+    config:
+      threshold: 4
+  file-lines:
+    config:
+      threshold: 250
```
Comment (givehug) [inline .codeclimate.yml]: This will be quite challenging for some *.test.js files at least.

**Comment by JeKachan** [comment]
I understand why needed similar-code, identical-code, but not understand why we need other restrictions (and where from they values where took) - not clear how they can improve code quality.

File: .codeclimate.yml (hunk 1 above)
Comment (andriichumak) [reply .codeclimate.yml]: Well, this is a default value, so it was already like this for some time. I agree that for tests it's too small, but for production code I think this is a reasonable value. We could add tests into ignore list. I'll see what config options there are.

**Comment by andriichumak** [comment]
@JeKachan those checks would help us not to be messy with our code. You can read about the concepts here and on adjusting pages:
https://docs.codeclimate.com/docs/cognitive-complexity

**Comment by andriichumak** [comment]
Values for those restrictions are based on default ones. If we will feel that some values are too strict or too loose - we'll adjust the config. For now it seams reasonable.

**Comment by andriichumak** [comment]
So, from now on we will not merge unless CodeClimate says it's OK. If some rules will start to bother you and you think they are unfair - we'll review those.

**Pull request merged**